examples/tree-view/actions/index.js

```diff
@@ -0,0 +1,26 @@
+export const INCREMENT = 'INCREMENT'
+export const CREATE_NODE = 'CREATE_NODE'
+export const ADD_CHILD = 'ADD_CHILD'
+
+export function increment(nodeId) {
+  return {
+    type: INCREMENT,
+    nodeId
+  }
+}
+
+let nextId = 0
+export function createNode() {
+  return {
+    type: CREATE_NODE,
+    nodeId: `new_${nextId++}`
+  }
+}
+
+export function addChild(nodeId, childId) {
```

Comment:
This is what I tried at first but it made the reducer a little bit awkward. It's hard to say which approach is cleaner until the app grows large enough and more use cases are implemented so I went with this version that keeps the reducers simpler.